tests/utils/executeQuery.js

```diff
@@ -2,7 +2,7 @@ const schema = require("../../schema");
 const { graphql } = require("graphql");
 
 async function executeQuery(query, args = {}, ctx = {}) {
-  return await graphql(schema, query, {}, ctx, args);
+    return await graphql(schema, query, {}, ctx, args); // eslint-disable-line no-return-await
```

Comment:
The rule was complaining of `async` without `await`. Shall I remove the async from the function too?